service/controller/v25/adapter/guest_launch_configuration.go

```diff
@@ -33,12 +34,40 @@ func (l *GuestLaunchConfigAdapter) Adapt(config Config) error {
 	l.WorkerImageID = config.StackState.WorkerImageID
 	l.WorkerAssociatePublicIPAddress = false
 
-	if config.StackState.WorkerDockerVolumeSizeGB <= 0 {
-		config.StackState.WorkerDockerVolumeSizeGB = defaultEBSVolumeSize
+	{
```

Comment:
The type bullshit here is just legacy. The handling replicates the defaulting but I am fairly certain it is not useful at this point. This code here will be cleaned up the more and more we phase out the old stack handlings. For now my goal with this is to ease the stack outputs handling in the new resources.